drivers/gles2/rasterizer_canvas_gles2.h

```diff
@@ -69,9 +133,28 @@ class RasterizerCanvasGLES2 : public RasterizerCanvas {
 
 		GLuint ninepatch_vertices;
 		GLuint ninepatch_elements;
-
 	} data;
 
+	struct BatchData {
+		GLuint GL_vertex_buffer;
```

Comment:
Other places we use lowercase "gl" when it is our own variable name. So "gl_vertex_buffer"